spring-boot-project/spring-boot-test/src/main/java/org/springframework/boot/test/context/SpringBootTestContextBootstrapper.java

```diff
@@ -265,7 +266,7 @@ private boolean containsNonTestComponent(Class<?>[] classes) {
 		return result;
 	}
 
-	private List<String> getAndProcessPropertySourceProperties(
+	protected List<String> getAndProcessPropertySourceProperties(
```

Comment:
If `org.springframework.boot.test.context.SpringBootTestContextBootstrapper.getProperties(Class<?>)` was overridden instead, I think this method could continue to be `private`. Perhaps I have missed something?